test/customValidators_test.js

```diff
@@ -300,7 +300,65 @@ describe('validateChannelParams()', function() {
     }`;
     const parsedInput = JSON.parse(inputString);
 
-    expect(() => validateChannelParams(parsedInput, inputString, input)).to.throw('Not all channel parameters are described with parameter object');
+    expect(() => validateChannels(parsedInput, inputString, input)).to.throw('Not all channel parameters are described with parameter object');
+  });
+
+  it('should successfully validate channel name without variable', async function() {
+    const inputString = `{
+      "channels": {
+        "test/test01": {
+          "parameters": {
+            "test": {
+              "schema": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    }`;
+    const parsedInput = JSON.parse(inputString);
+
+    expect(validateChannels(parsedInput, inputString, input)).to.equal(true);
+  });
+
+  it('should throw error that one of provided channel name is invalid', async function() {
+    const inputString = `{
+      "channels": {
+        "/user/signedup?foo=1": {
+          "subscribe": {
```

Comment:
same as with the other test case, I think you can easily just pass an empty channel object